<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>990</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>990</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>321</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>651</width>
      <height>541</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="MessageBoard" name="msgBoard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>559</y>
      <width>641</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="ButtonBar" name="b_widget" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>560</y>
      <width>325</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageBoard</class>
   <extends>QPlainTextEdit</extends>
   <header>messageboard.h</header>
  </customwidget>
  <customwidget>
   <class>ButtonBar</class>
   <extends>QWidget</extends>
   <header>buttonbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
